server/channels/api4/channel_bookmark.go

```diff
@@ -196,3 +197,95 @@ func updateChannelBookmark(c *Context, w http.ResponseWriter, r *http.Request) {
 		c.Logger.Warn("Error while writing response", mlog.Err(err))
 	}
 }
+
+func updateChannelBookmarkSortOrder(c *Context, w http.ResponseWriter, r *http.Request) {
+	if c.App.Channels().License() == nil {
+		c.Err = model.NewAppError("updateChannelBookmarkSortOrder", "api.channel.bookmark.channel_bookmark.license.error", nil, "", http.StatusForbidden)
+		return
+	}
+
+	connectionID := r.Header.Get(model.ConnectionId)
+
+	c.RequireChannelId()
+	if c.Err != nil {
+		return
+	}
+
+	var newSortOrder int64
+	if err := json.NewDecoder(r.Body).Decode(&newSortOrder); err != nil {
```

Comment:
Are you expecting just a number in the body? No key/value or something similar? Can you expand on the reason?

Reply:
The endpoint only requires one piece of information, and due its format that's unlikely to change. I'm 1/5 on this tho, so if we feel strongly about having a JSON object, I'm happy to update the PR

Reply:
No strong opinions just wanted to know